differ/differ.go

```diff
@@ -244,6 +244,26 @@ func findRuleByNameAndErrorKey(
 	return
 }
 
+// evaluateFilterExpression function tries to evaluate event filter expression
+// based on provided threshold values and actual reccomendation values
+func evaluateFilterExpression(eventFilter string,
+	likelihoodThreshold int, impactThreshold int, severityThreshold int, totalRiskThreshold int,
```

Comment:
all these "int" can be merged into just one at the end of the parenthesis

Reply:
fixed